Neos.Neos/Classes/Fusion/Helper/BackendUserHelper.php

```diff
@@ -0,0 +1,44 @@
+<?php
+namespace Neos\Neos\Fusion\Helper;
+
+/*
+ * This file is part of the Neos.Neos package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Eel\ProtectedContextAwareInterface;
+use Neos\Flow\Annotations as Flow;
+use Neos\Neos\Service\UserService;
+
+/**
+ * BackendUser helper for translations in the backend
+ */
+class BackendUserHelper implements ProtectedContextAwareInterface
+{
+    #[Flow\Inject(lazy: false)]
+    protected UserService $userService;
+
+    /**
+     * The interface language the user selected
+     * Falls back to the default language defined in the settings
+     *
+     * Example::
+     *
+     *     Translation.id("mh").locale(Neos.BackendUser.interfaceLanguage()).translate()
+     *
+     */
+    public function interfaceLanguage(): string
```

Comment:
The docs should be adjusted a bit to explain what kind of string is returned. As the method is called language one could expect `German`, but apparently it is a locale like `de_de` if it can be used with the `Translation` helper.

Reply:
adjusted ;)

sadly we cannot return a real `Flow\I18N\Locale` as the Translation helper cannot handle it / wont even cast it.